data/gegno/gegno intro missions.txt

```diff
@@ -432,6 +433,312 @@ mission "Giaru Gegno: Revisit"
 				decline
 
 
+mission "Suspicious Scin"
+	invisible
+	landing
+	source "Giaru Gegno"
+	destination "Yiia Iyr"
+	to offer
+		has "Visit Quarg in Gegno Space: offered"
+		not "First Contact: Gegno Scin: offered"
+		not "Gegno Genocide Defense: offered"
+	on enter
+		system
+			near "Aleit" 4 100
+		conversation
+			action
+				set "gegno: mysterious device"
+			`As you're traveling through Gegno space, a faint signal starts emanating from somewhere in your ship. You aren't sure what it is, as it doesn't appear to originate from the ship itself, so you decide to take a look. After a few minutes of searching for the culprit, you come across a device not too dissimilar from the one that the Gegno you met on Var Oti had given you, arbitrarily shoved into some corner of your ship's infrastructure. You don't know how or when it got there, but at least it wasn't something more dangerous like an explosive.`
+			`	Once you retrieve the device, you notice that alongside with the signal it's emitting, there is some sort of map display that has several locations in the region marked in some way or another, along with various scribblings in a language you can't understand. Although it's hard to make sense of what any of it means, one particular planet displayed has several graphics around it, much more extravagant than the rest. Perhaps finding this planet will shed some light on this strange situation.`
```

Comment:
```suggestion
			`	Once you retrieve the device, you notice that along with the signal it's emitting, there is some sort of map display that has several locations in the region marked in some way or another, along with various scribblings in a language you can't understand. Although it's hard to make sense of what any of it means, one particular planet displayed has several graphics around it, much more extravagant than the rest. Perhaps finding this planet will shed some light on this strange situation.`
```
You could also do `that alongside the signal` or `that with the signal`. Up to you!